pkg/sdn/plugin/pod.go

```diff
@@ -0,0 +1,453 @@
+package plugin
```

Comment:
Please follow the godoc conventions - always start with the method name, complete sentences.  General style applies to all our code.  Interfaces should be documented as well.

Field godoc should be `// fieldName does blah to blarg`